.gitignore

```diff
@@ -75,7 +75,6 @@ jarjar-rules.txt
 /android/app/fabric.properties
 /ios/Exponent/Generated/EXKeys.h
 /ios/ExponentIntegrationTests/EXTestEnvironment.plist
-/ios/Podfile
```

Comment:
revert all this, your git branch looks like it is in a bad state

Reply:
yeah, made a mistake and rebased from master